### ISSUE TITLE
Credits

### ISSUE THREAD

**Issue by NotSoMik** [andy-emma/utils#1]
Credits where? who gave you the authorization to publish his codes? At least read the license

**Comment by GioGio03** [comment]
You are right lmao, this guy is a loser and in the package.json he put: "author": "Spirit | Comunity & Devlopment", but you can clearly see that the author is leaf lmao, delete this project because you are not the author
